src/components/tool/guide/Config/ConfigDuration.js

```diff
@@ -0,0 +1,104 @@
+import { CheckCircleIcon, WarningIcon } from "@chakra-ui/icons";
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  CloseButton,
+  Flex,
+  Heading,
+  Square,
+  Tooltip,
+} from "@chakra-ui/react";
+
+import { useState } from "react";
+
+import InputNumberStepper from "../../../input/InputNumberStepper";
+
+export default function ConfigDuration({
+  title,
+  description,
+  selected,
+  onChange,
+  value,
+  inputPropsHour,
+  inputPropsMinute,
+  showAlert = false,
+  alertDismissible = true,
+  alertType = "info",
+  alertMsg = "",
+}) {
+  const [alertDismissed, setAlertDismissed] = useState(false);
+  const onClose = () => {
+    setAlertDismissed(true);
+  };
+
+  return (
+    <Box>
+      <Flex direction="row" width="100%" align="center">
+        <Box w="120px">
+          {selected ? (
+            <CheckCircleIcon color="green.500" />
+          ) : (
+            <Tooltip label="Pick an option">
+              <WarningIcon color="orange.500" />
+            </Tooltip>
+          )}
+          <Tooltip label={description} placement="top" hasArrow>
+            <Heading size="xs" mb="0" display="inline" ml="2">
+              {title}
+            </Heading>
+          </Tooltip>
+        </Box>
+        <Square size="10px" />
+        <Flex>
+          <InputNumberStepper
```

Comment:
One last thing I noticed, can we allow incrementing using the mouse wheel?

https://v2.chakra-ui.com/docs/components/number-input#increment-value-using-mouse-wheel

Reply:
Ticket created: https://eresearchqut.atlassian.net/browse/ERP-2008